Makefile

```diff
@@ -297,21 +262,25 @@ build-binaries-linux:
 
```

Comment:
is `GOARCH=amd64` missing above?

Reply:
Yes, although missing `GOARCH=amd64` here will not affect anything, but having this is better. I have updated.